src/tests/Directory.Build.targets

```diff
@@ -22,6 +22,10 @@
     <NoWarn Condition="'$(IsMergedTestRunnerAssembly)' == 'true'">$(NoWarn);CS2008</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <IlcCompileDependsOn Condition="'$(CustomIlcCompileDependsOn)' == 'true'">Compile;ComputeIlcCompileInputs;SetupOSSpecificProps;PrepareForILLink</IlcCompileDependsOn>
```

Comment:
What does this do? Looking at how we set this up, don't we end up with the same things?

https://github.com/dotnet/runtime/blob/05e25ff8c3a623df74720f430c270eba0f1d41b0/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L97-L102